### PR TITLE
Add changeset and support for package releasing

### PR DESCRIPTION
**Pull request by satyamakgec**
Closes: #91

## Description
Allow publishing a package to the npm using the changeset.

- Any changes merged in the master will lead to the release of a new version of the `@onflow/flow-nft` package.
______

For contributor use:

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/flow-nft/blob/master/CONTRIBUTING.md#styleguides).
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "url": "git+https://github.com/onflow/flow-nft.git"
+  },
+  "keywords": [
+    "flow-nft",
+    "nft",
+    "NonFungibleToken",
+    "flow"
+  ],
+  "author": "onFlow",
+  "license": "MIT",
```
Comment (alilloig) [inline package.json:25]: This license is referring to the npm package or to the nft repo? just because the flow-nft is unlicensed rather than under MIT

**Review by alilloig — APPROVED** [review]
looks good!!

**Review by joshuahannan — COMMENTED** [review]
What exactly does this do? I'm not very familiar with this

**Comment by alilloig** [comment]
If I understood it well, when any changes are merged into master, this will automatically update the npm package right?

**Review by joshuahannan — APPROVED** [review]

**Pull request merged**